diff --git a/node_modules/@jspm/generator/dist/generator-9582750c.js b/node_modules/@jspm/generator/dist/generator-9582750c.js
index 4826875..13a5940 100644
--- a/node_modules/@jspm/generator/dist/generator-9582750c.js
+++ b/node_modules/@jspm/generator/dist/generator-9582750c.js
@@ -217,6 +217,7 @@ async function createTsAnalysis(source, url) {
             sourceType: 'module',
             parserOpts: {
                 plugins: [
+                    'decorators',
                     'jsx'
                 ],
                 errorRecovery: true
@@ -225,6 +226,7 @@ async function createTsAnalysis(source, url) {
                 [
                     babelPresetTs,
                     {
+                        allowDeclareFields: true,
                         onlyRemoveTypeImports: true
                     }
                 ]
@@ -3721,9 +3723,15 @@ class TraceMap {
    * @param {} parentUrl URL of the parent context for the specifier.
    * @param {} seen Cache for optimisation.
    */ async visit(specifier, opts, parentUrl = this.baseUrl.href, seen = new Set()) {
-        var _this_opts_ignore;
         if (!parentUrl) throw new Error('Internal error: expected parentUrl');
-        if ((_this_opts_ignore = this.opts.ignore) === null || _this_opts_ignore === void 0 ? void 0 : _this_opts_ignore.includes(specifier)) return;
+        if (Array.isArray(this.opts.ignore)) {
+            for (const specOrRegexp of this.opts.ignore) {
+                if ((specOrRegexp instanceof RegExp && specOrRegexp.test(specifier))
+                    || typeof specOrRegexp === 'string' && specifier === specOrRegexp) {
+                    return;
+                }
+            }
+        }
         if (seen.has(`${specifier}##${parentUrl}`)) return;
         seen.add(`${specifier}##${parentUrl}`);
         this.log('tracemap/visit', `Attempting to resolve ${specifier} to a module from ${parentUrl}, toplevel=${opts.toplevel}, mode=${opts.installMode}`);
diff --git a/node_modules/@jspm/generator/lib/generator.d.ts b/node_modules/@jspm/generator/lib/generator.d.ts
index c81178c..ee12d95 100644
--- a/node_modules/@jspm/generator/lib/generator.d.ts
+++ b/node_modules/@jspm/generator/lib/generator.d.ts
@@ -246,7 +246,7 @@ export interface GeneratorOptions {
      * await generator.install("@react-three/fiber@7")
      * ```
      */
-    ignore?: string[];
+    ignore?: (string|RegExp)[];
     /**
      * Lockfile data to use for resolutions
      */
