diff --git a/node_modules/@lit-labs/eleventy-plugin-lit/worker/worker.js b/node_modules/@lit-labs/eleventy-plugin-lit/worker/worker.js
index 9d5a09f..089e1dc 100644
--- a/node_modules/@lit-labs/eleventy-plugin-lit/worker/worker.js
+++ b/node_modules/@lit-labs/eleventy-plugin-lit/worker/worker.js
@@ -9,6 +9,25 @@ import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 if (parentPort === null) {
     throw new Error('worker.js must only be run in a worker thread');
 }
+
+export class ObserverShim {
+  observe() { void 0; }
+
+  disconnect() { void 0; }
+}
+
+globalThis.window ??= globalThis;
+globalThis.ErrorEvent ??= Event;
+globalThis.IntersectionObserver ??= ObserverShim;
+globalThis.MutationObserver ??= ObserverShim;
+globalThis.getComputedStyle ??= function() {
+  return {
+    getPropertyPriority() { return ''; },
+    getPropertyValue() { return ''; },
+  };
+};
+
+
 let initialized = false;
 parentPort.on('message', async (message) => {
     switch (message.type) {
