diff --git a/node_modules/@patternfly/pfe-tools/dev-server/plugins/import-map-generator.js b/node_modules/@patternfly/pfe-tools/dev-server/plugins/import-map-generator.js
index ad61e24..40579d3 100644
--- a/node_modules/@patternfly/pfe-tools/dev-server/plugins/import-map-generator.js
+++ b/node_modules/@patternfly/pfe-tools/dev-server/plugins/import-map-generator.js
@@ -92,6 +92,8 @@ export function importMapGeneratorPlugin(options) {
                 inputMap: options?.inputMap,
                 customProviders: { monorepotypescript },
                 resolutions: options?.resolutions,
+                typeScript: options?.typeScript,
+                ignore: options?.ignore,
             }), options));
         },
     };
