diff --git a/node_modules/@patternfly/pfe-tools/11ty/plugins/custom-elements-manifest.cjs b/node_modules/@patternfly/pfe-tools/11ty/plugins/custom-elements-manifest.cjs
index 03b4fee..c7465d8 100644
--- a/node_modules/@patternfly/pfe-tools/11ty/plugins/custom-elements-manifest.cjs
+++ b/node_modules/@patternfly/pfe-tools/11ty/plugins/custom-elements-manifest.cjs
@@ -1,5 +1,4 @@
 // @ts-check
-const { EleventyRenderPlugin } = require('@11ty/eleventy');
 const { join } = require('node:path');
 const { existsSync } = require('node:fs');
 const { glob } = require('glob');
@@ -13,8 +12,6 @@ const isDir = dir => stat(dir).then(x => x.isDirectory, () => false);
  * @param {import('./types').PluginOptions} pluginOpts
  */
 module.exports = function configFunction(eleventyConfig, pluginOpts = {}) {
-  eleventyConfig.addPlugin(EleventyRenderPlugin);
-
   eleventyConfig.addGlobalData('env', () => process.env);
 
   eleventyConfig.addGlobalData('demos', async function demos() {
diff --git a/node_modules/@patternfly/pfe-tools/package.json b/node_modules/@patternfly/pfe-tools/package.json
index 9d19722..bdb26d9 100644
--- a/node_modules/@patternfly/pfe-tools/package.json
+++ b/node_modules/@patternfly/pfe-tools/package.json
@@ -63,7 +63,7 @@
     "**/*.{js,cjs,d.ts,map}"
   ],
   "peerDependencies": {
-    "@11ty/eleventy": "^2.0.1",
+    "@11ty/eleventy": "^2.0.1 || ^3.0.0",
     "@11ty/eleventy-plugin-directory-output": "^1.0.1",
     "@11ty/eleventy-plugin-syntaxhighlight": "^5.0.0",
     "@changesets/cli": "^2.27.1",
