diff --git a/node_modules/eslint-plugin-lit-a11y/lib/utils/hasAccessibleChildren.js b/node_modules/eslint-plugin-lit-a11y/lib/utils/hasAccessibleChildren.js
index 66186f9..d9cffaf 100644
--- a/node_modules/eslint-plugin-lit-a11y/lib/utils/hasAccessibleChildren.js
+++ b/node_modules/eslint-plugin-lit-a11y/lib/utils/hasAccessibleChildren.js
@@ -6,7 +6,7 @@ const { isHiddenFromScreenReader } = require('./isHiddenFromScreenReader.js');
  */
 function isElement(node) {
   // NB: this isn't accurate, but suffices for use in `hasAccessibleChildren`.
-  return node.type !== 'text';
+  return node.type !== 'text' && node.type !== 'comment';
 }
 
 /**
